io.github.lavenderdotpet.LibreQuake.yml

```diff
@@ -0,0 +1,62 @@
+app-id: io.github.lavenderdotpet.LibreQuake
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: librequake.sh
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+
+modules:
+  - shared-modules/libmad/libmad.json
+
+  - name: quakespasm
+    sources:
+      - type: git
+        url: https://github.com/sezero/quakespasm.git
+        tag: quakespasm-0.96.3
+    buildsystem: simple
+    build-commands:
+      - make --directory=Quake USE_SDL2=1 LDFLAGS="-L/app/lib"
```

Comment:
Use `make-args`, drop `buildsystem: simple`, move the two install commands to `post-install`

Reply:
If I do this. Assuming I understand this correctly. I get 
```
Error: module quakespasm: Can't find autogen, autogen.sh or bootstrap
```
I assume by default the autotools build system is used. This is just a simple makefile project though. The other buildsystems don't seem to fit either. What do?

Reply:
Set `no-autogen: true`

Reply:
I got it to work.